src/components/vr_cooperation/include/vr_cooperation/vr_module.h

```diff
@@ -142,29 +172,18 @@ class VRModule : public functional_modules::GenericModule,
       application_manager::MessagePtr msg);
 
   /**
-   * @brief handle HMI message
-   * @param msg HMI message
-   */
-  functional_modules::ProcessResult HandleHMIMessage(
-      application_manager::MessagePtr msg);
-
-  /**
    * @brief Set mobile message type
    * @param msg mobile message
    */
   bool SetMessageType(application_manager::MessagePtr& msg) const;
 
-  /**
-   * @brief Set HMI message type
-   * @param msg hmi message
-   */
-  bool SetHMIMessageType(application_manager::MessagePtr& msg) const;
-
   static const functional_modules::ModuleID kVRModuleID = 154;
   functional_modules::PluginInfo plugin_info_;
   request_controller::RequestController request_controller_;
 
   VRProxy proxy_;
+  uint32_t activated_connection_key_;
+  uint32_t stored_app_id_;
```

Comment:
This variable use to determine what is default app.
I think it should be default_app_id_